olaaf_django/messages.py

```diff
@@ -0,0 +1,25 @@
+# TODO: i18n
+from .utils import format_date
+
+VALID_OUTDATED_HTML_DOC_MSG = """
+Document was valid between {start_date} and {end_date}.
+"""
+
+VALID_OUTDATED_PDF_DOC_MSG = """
```

Comment:
Will we use these messages in production?

Reply:
We will. These apply only to HTML files.

Reply:
Some of them don't sound a little off to me. Like "is valid since" -> "has been valid since", "is no longer present" -> "no longer exists" maybe? Should we ask @dgreisen and/or @JasonRJFleischer ?